bin/PullAndProcessGsAnalysisData.sh

```diff
@@ -257,27 +292,30 @@ function mergeSamplesheets(){
 	local _captkit
 	for i in "${uniqProjects[@]}"
 	do
+		sampie="${TMP_ROOT_DIR}/${_batch}/${i}.csv"
+		cp "${sampie}"{,.converted}
+		printf '\n'     >> "${sampie}.converted"
+		sed -i 's/\r/\n/g' "${sampie}.converted"
+		sed -i "/^[\s${SAMPLESHEET_SEP}]*$/d" "${sampie}.converted"
+		mv "${sampie}.converted" "${sampie}"
+		
 		if [[ "${teller}" -eq '0' ]]
 		then
 			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Creating new combined samplesheet for the processing of the Analysis data"
 			#Renaming samplesheet to solely the GS_XX (e.g. GS_182.. so without the A,B,C etc suffix)
 			_projectName=$(echo "${i}" | grep -Eo 'GS_[0-9]+')
 			_captkit=$(echo "${i}" | awk 'BEGIN {FS="-"}{print $NF}')
```

Comment:
`${_capturingKit}` would follow the naming convention for variable names in this repo.

Reply:
?

Reply:
fixed